test/automation/src/sql/notebook.ts

```diff
@@ -78,6 +78,14 @@ export class Notebook {
 		const outputComponent = '.notebook-cell.active .notebook-output';
 		await this.code.waitForElement(outputComponent);
 	}
+
+	async trustNotebook(): Promise<void> {
+		await this.toolbar.trustNotebook();
+	}
+
+	async isTrusted(): Promise<void> {
```

Comment:
Why is this returning void? With a name like `isTrusted` I would expect this to return a boolean value saying whether it's trusted or not. 

If the intent here is just to wait until it's set to be trusted then maybe `waitUntilTrusted` or something similar would be  a better name.

Reply:
I changed the method name to waitForTrustedIcon (and added another method waitForNotTrustedIcon).